### ISSUE TITLE
[Bug]: Bugged network list on Sonoma 14.0

### ISSUE THREAD

**Issue by yurixander** [OpenIntelWireless/HeliPort#247]
### What happened?

When the OS starts and I click on the Heliport icon (or when I open Heliport for the first time by clicking on its icon after it has been launched), the network list is all messed up and shrinked.

If I click on it a few times it seems to get better, but never fully 100%. Perhaps it may be due to changes on macOS 14.0's constraint system (ie. Swift code)?

![bug preview](https://github.com/OpenIntelWireless/HeliPort/assets/101931215/0e4631c9-c015-4547-aeb5-e85ef17fff31)
_(I removed my WiFi's name for privacy)_

### Steps To Reproduce

1. Install Sonoma 14.0
2. Open Heliport by clicking on its icon on the top bar
3. The network list should be all bugged out

### Software Version

v1.4.1 AND v1.5.0-alpha

### OS Version

14.0

### Relevant Log Output

Using `log show --predicate 'process == "HeliPort" && (eventType == "Info" || eventType == "Debug" || eventType == "Error")'`:

```
Filtering the log data using "process == "HeliPort" AND (type == "Info" OR type == "Debug" OR type == "Error")"
Skipping info and debug messages, pass --info and/or --debug to include.
Timestamp                       Thread     Type        Activity             PID    TTL
--------------------------------------------------------------------------------------------------------------------
Log      - Default:          0, Info:                0, Debug:             0, Error:          0, Fault:          0
Activity - Create:           0, Transition:          0, Actions:           0
```

**Comment by yurixander** [comment]
**Update:** I've cloned and built changes from #246, and the UI issues seem to be resolved:

![fixed preview](https://github.com/OpenIntelWireless/HeliPort/assets/101931215/f200c4d9-968e-4a39-9b5a-5a15a7b9c1ae)

**Comment by Engredaesm** [comment]
could you share your release please

> **Update:** I've cloned and built changes from #246, and the UI issues seem to be resolved:
> 
> ![fixed preview](https://user-images.githubusercontent.com/101931215/270865657-f200c4d9-968e-4a39-9b5a-5a15a7b9c1ae.png)

could you share your release please

**Comment by mithleshjs** [comment]
@Engredaesm  https://github.com/diepeterpan/HeliPort/releases/download/v1.5.0.e/HeliPort.working.-.Launch.At.Login.zip

**Comment by Engredaesm** [comment]
> @Engredaesm https://github.com/diepeterpan/HeliPort/releases/download/v1.5.0.e/HeliPort.working.-.Launch.At.Login.zip

Thank you

**Comment by diepeterpan** [comment]
This pull request will resolve the Sonoma problems.

https://github.com/OpenIntelWireless/HeliPort/pull/246

**Comment by perez987** [comment]
@diepeterpan 
Thank you, last changes made by you have fixed this issue.

**Comment by perez987** [comment]
@diepeterpan 
`HeliPort.working.-.Launch.At.Login.zip` includes the improvements of `HeliPort_more_Sonoma_like.zip`? 
What app must be selected?
Thanks.

**Comment by diepeterpan** [comment]
> @diepeterpan `HeliPort.working.-.Launch.At.Login.zip` includes the improvements of `HeliPort_more_Sonoma_like.zip`? What app must be selected? Thanks.

Either one will work, but if you want something that looks a bit more like Sonoma then you can use the second one which is based on the **'sonoma-like'** of the code.  

Once the pull request for Sonoma fixes **'master branch'** is accepted then I will log the pull request for the **'sonoma-like'** branch. Apparently, the owner of the Heliport repo is very busy and these are hanging in limbo :-(

**Comment by perez987** [comment]
@diepeterpan 
Thank you!!!

**Comment by toannxm** [comment]
Is there any chance to fix this issue? @diepeterpan

**Comment by perez987** [comment]
@toannxm 
The issue has been fixed 2 months ago. Get the latest release.

**Comment by williambj1** [comment]
Fixed in v1.5.0
